### PR TITLE
Adding Zauba cloud as gold sponser

### PR DESCRIPTION
**Pull request by Rohithgilla12**
* **Please check if the PR fulfills these requirements**

- [x] Screenshots for changes (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:
Adding Zauba as gold sponser

### CODE REVIEW

**Comment by Rohithgilla12** [comment]
#185

File: content/blog/announcing_zauba_as_gold_sponsor.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<p class="text-center">
+	<a href="https://zauba.company/" target="_blank">
+		<img src="https://raw.githubusercontent.com/pythonindia/inpycon2018/master/img/sponsors/zaubacloud.png" alt="Zauba Cloud" height="200"/>
+	</a>
+</p>
+
+Hi Everyone, it is our pleasure to announce Zauba Cloud, as PyCon India 2018’s gold sponsor.
+<!-- PELICAN_END_SUMMARY -->
+
+Zauba Cloud is a next generation cloud computing platform that accelerates product development,reduces cost and increases overall performance and reliability of assets on cloud
```
Comment (ananyo2012) [inline content/blog/announcing_zauba_as_gold_sponsor.md]: @Rohithgilla12 Add the link in the company name here as well

**Comment by ananyo2012** [comment]
@realslimshanky Yup they gave a short description

**Comment by vipulgupta2048** [comment]
Hey, my server is throwing some errors (#188) as I was proceeding to fix some small problems with the blog. 
I have opened issues for the same that could be implemented in this PR only. Do take a look #189

File: content/blog/announcing_zauba_as_gold_sponsor.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Title: Announcing Zauba as Gold Sponsor
+Date: 2018-09-29 13:30
```
Comment (ananyo2012) [inline content/blog/announcing_zauba_as_gold_sponsor.md]: Give todays date here and time after the latest blog post

**Comment by ananyo2012** [comment]
@Rohithgilla12 Can you update this by today ?

**Comment by Rohithgilla12** [comment]
Yes sure, will do in 20 min

**Comment by Rohithgilla12** [comment]
Done @ananyo2012

**Comment by ananyo2012** [comment]
Can you send a screenshot to just verify ?

**Comment by Rohithgilla12** [comment]
I'm in college now, can send you in 2 hours

**Comment by ananyo2012** [comment]
Sure no problem

**Comment by Rohithgilla12** [comment]
screenshot 2018-10-01 18 46 11
Sorry for the delay

**Pull request merged**